scripts/build.sh

```diff
@@ -4,7 +4,7 @@ PROJECT_ROOT="$(dirname "$(dirname "$(readlink -f "$0")")")"
 cd "${PROJECT_ROOT}" || exit 1
 
 if [[ -n "${ALL}" ]]; then
-	GOBIN="${PROJECT_ROOT}" go install ./cmd/...
+	GOBIN="${PROJECT_ROOT}" go install -tags prod ./cmd/...
 else
-	GOBIN="${PROJECT_ROOT}" go install ./cmd/swapd ./cmd/swapcli
+	GOBIN="${PROJECT_ROOT}" go install -tags prod ./cmd/swapd ./cmd/swapcli
```

Comment:
Test code from the `monero` package won't get compiled into the binary if you pass the `prod` tag. If you don't care, you can build the binaries without using the `prod` build tag.